packages/node/src/lib/messages/messages.ts

```diff
@@ -0,0 +1,26 @@
+import { IMessagesPayload, IMessages } from './messages.interface';
+
+import { WithHttp } from '../novu.interface';
+
+const BASE_PATH = '/messages';
+
+export class Messages extends WithHttp implements IMessages {
+  async list(data?: IMessagesPayload) {
+    const queryParams: Partial<IMessagesPayload> & {
+      transactionId?: string[];
+    } = {};
+    data?.page && (queryParams.page = data?.page);
+    data?.limit && (queryParams.limit = data?.limit);
+    data?.subscriberId && (queryParams.subscriberId = data?.subscriberId);
+    data?.channel && (queryParams.channel = data?.channel);
+    data?.transactionIds && (queryParams.transactionId = data?.transactionIds);
```

Comment:
I think we need to try this and see that the HTTP call generated by the Node package for multiple transactionIds creates the query params in the shape you defined them: `?transactionId=val1&transactionId=val2`.
Because I am afraid Axios is going to generate the call like this:
`?transactionId[]=val1&transactionId[]=val2`.

Reply:
Yeah you are correct, 
but `?transactionId[]=val1&transactionId[]=val2` pattern has no effect at the controller level, nestjs query converts this also to `["val1" ,"val2"]`  as same as it converts `?transactionId=val1&transactionId=val2`